airflow/api_fastapi/core_api/datamodels/dag_run.py

```diff
@@ -96,18 +96,16 @@ def check_data_intervals(cls, values):
             )
         return values
 
+    ## when logical date is null, the run id should be generated from run_after + random string.
+    # TODO we need to modify this validator after https://github.com/apache/airflow/pull/46398 is merged
```

Comment:
```suggestion
    ## when logical date is null, the run id should be generated from run_after + random string.
    # TODO: AIP83: we need to modify this validator after https://github.com/apache/airflow/pull/46398 is merged
```

Just so we can find this easier later.

Reply:
Implemented